### ISSUE TITLE
Fullscreen programs are forced into a window but still think they're fullscreened

### ISSUE THREAD

**Issue by ngoomie** [NsCDE/NsCDE#91]
When attempting to launch games in fullscreen mode while using NsCDE, they get forced into a window, yet still seemingly think they're in fullscreen mode. Clickable objects in a game will also be offset from where they're actually displayed. I've had this happen with two games, Sid Meier's Civilization VI and Team Fortress 2, which are the only two games I've attempted to play while using NsCDE. Both of them are native Linux games, and I'm running them + NsCDE on EndeavourOS, with NsCDE being installed from the `nscde-git` package in the AUR.

Civ VI forced itself to the smallest available windowed resolution, but still was expecting menu objects to be where they would be if playing in fullscreen, which made it impossible to change the actual video settings to match as the apply button is expected to be outside of the window. It's possible to play Civ VI no problem if you first switch it to windowed mode at the appropriate resolution in another desktop environment, then come back to NsCDE and launch it again, so it is indeed only a problem if the game expects to be able to launch in fullscreen mode.

Team Fortress 2 half attempts to force itself to the lowest possible windowed resolution while still outputting in whatever your most recently used resolution is, causing the video to become squashed and/or stretched to fit the window. It's possible to access the video settings menu despite TF2 also experiencing the menu offsetting issue, but when in this state it doesn't respond to attempting to change and apply any graphics settings. The game works just fine if you use the `-w`, `-h` and `-windowed` launch commands in Steam to force the game to launch with a custom resolution and in an actual window, so like with Civ VI, this definitely only is a problem if the game is assuming it can launch in fullscreen mode.

Sorry if this is worded poorly, it's a really unusual issue so I'm having a hard time explaining what's even happening. I wanted to try and be as thorough as possible. Thank you very much in advance!

**Comment by NsCDE** [comment]
Hi.

This probably does not anything to do with NsCDE as such, but maybe fvwm.

Tell me more:
- NsCDE version you use
- FVWM version NsCDE uses
- Have you tried to go into full screen mode with Meta+Alt+F?

**Comment by ngoomie** [comment]
I'm using NsCDE 2.0-1, fvwm 2.6.9, and both games are unresponsive to Meta+Alt+F.

**Comment by NsCDE** [comment]
Unfortunately, I don't play or have any games. I'm not sure if I put any of this what you mentioned in virtual machine that it will run at least for having debug environment. I didn't have any problems with fullscreening programs till now. I will try something.

In the meantime, have you tried to run this in plain FVWM 2.6.9 environment without NsCDE? Does this makes any difference?

**Comment by ngoomie** [comment]
Turns out this is indeed an FVWM issue, super sorry to have wasted your time. Testing in plain FVWM should've been one of the first things I tried before submitting a bug report.

Here's an issue on the FVWM Github that goes over a similar issue, in case anyone else runs into this in the future. Theirs was with Windows games running under Proton versus mine with native Linux games, but the solution seems like it should be applicable to this situation anyways. Haven't tested it myself, however.
https://github.com/fvwmorg/fvwm3/issues/574
